packages/forms-web-app/src/middleware/check-debug-allowed.js

```diff
@@ -0,0 +1,12 @@
+const config = require('../config');
+
```

Comment:
Nice, I like this approach. The only other thing I'd think about is if we always want to disallow debug mode on production environments. For example if domain contains myproddomain.com then prevent debug mode being enabled even if it's set to true. We could make that an application configuration value and support an array of domains as well, just so we have an additional mechanism in place to prevent potentially enabling debug mode in production.

Reply:
Seems sensible, might try and do that in a separate PR at some point